model/office/trigger.go

```diff
@@ -0,0 +1,111 @@
+package office
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"net/url"
+	"strings"
+
+	"github.com/cozy/cozy-stack/model/instance"
+	"github.com/cozy/cozy-stack/model/job"
+	jwt "github.com/golang-jwt/jwt/v4"
+	"github.com/labstack/echo/v4"
+)
+
+// SendSaveMessage is used by the trigger for asking OO to save the document in
+// the Cozy.
+type SendSaveMessage struct {
+	Key string `json:"key"`
+}
+
+func setupTrigger(inst *instance.Instance, key string) error {
+	sched := job.System()
+	infos := job.TriggerInfos{
+		Type:       "@every",
+		WorkerType: "office-save",
+		Arguments:  "10m",
+	}
+	msg := &SendSaveMessage{Key: key}
+	t, err := job.NewTrigger(inst, infos, msg)
+	if err != nil {
+		return err
+	}
+	return sched.AddTrigger(t)
```

Comment:
Shouldn't we check if a trigger already exists to avoid creating multiple triggers in case a document is opened in multiple tabs?